Vic2ToHoI4/Source/OutHoi4/CountryCategories/OutCountryCategory.cpp

```diff
@@ -4,11 +4,6 @@
 
 void HoI4::outputCountryCategory(std::ostream& out, const std::string& category, const TagsAndExtras& tagsAndExtras)
 {
-	if (tagsAndExtras.empty())
```

Comment:
Huh, this bit seemed logical.

Reply:
Turns out we want to output even the empty ones, else other scripting breaks and generates ~1000 errors.